suites/squid/nvmeof/tier-2_8-1_nvmeof_gateway_operations.yaml

```diff
@@ -1,70 +1,70 @@
 # Basic Ceph-NvmeoF sanity Test suite
 # cluster configuration file: conf/squid/nvmeof/ceph_nvmeof_sanity.yaml
-# Inventory: conf/inventory/rhel-9.3-server-x86_64-xlarge.yaml
+# Inventory: conf/inventory/rhel-9.5-server-x86_64-xlarge.yaml
 
 tests:
 # Set up the cluster
-  - test:
-      abort-on-fail: true
-      module: install_prereq.py
-      name: install ceph pre-requisites
-  - test:
-      abort-on-fail: true
-      config:
-        verify_cluster_health: true
-        steps:
-          - config:
-              command: bootstrap
-              service: cephadm
-              args:
-                mon-ip: node1
-                registry-url: registry.redhat.io
-                allow-fqdn-hostname: true
-                log-to-file: true
-          - config:
-              command: add_hosts
-              service: host
-              args:
-                attach_ip_address: true
-                labels: apply-all-labels
-          - config:
-              command: apply
-              service: mgr
-              args:
-                placement:
-                  label: mgr
-          - config:
-              command: apply
-              service: mon
-              args:
-                placement:
-                  label: mon
-          - config:
-              command: apply
-              service: osd
-              args:
-                all-available-devices: true
-      desc: RHCS cluster deployment using cephadm
-      destroy-cluster: false
-      module: test_cephadm.py
-      name: deploy cluster
+#   - test:
```

Comment:
Are these tests commented intentionally to allow simplification and compaction norms?